src/lib.rs

```diff
@@ -22,7 +23,7 @@
 //! [v0.4.0](https://rgbds.gbdev.io/docs/v0.4.0/rgbds.5)   | v9 r3
 //! [v0.3.10](https://rgbds.gbdev.io/docs/v0.3.10/rgbds.5) | v6
 //!
-//! Earlier releases use v6, up to and including v0.3.4.
+//! RGBDS releases v0.3.4 and earlier use object format v6.
```

Comment:
I think this was rather meaning to say that v6 lasted *from* 0.3.4, until 0.3.10.